buildSrc/src/main/kotlin/scripts/variants.gradle.kts

```
@@ -48,14 +48,15 @@ android {
             isMinifyEnabled = false
             applicationIdSuffix = ".${BuildTypes.DEBUG}"
             isDebuggable = true
-            if(enableSigning)
+            if (isMinifyEnabled) proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
```

Maybe add a comment here, justifying the possible addition of proguardFiles for `debug` builds?

```suggestion
            // Just in case a developer is trying to debug some prod crashes by turning on minify
            if (isMinifyEnabled) proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
```